amundsen_application/static/js/ducks/dashboard/types.ts

```diff
@@ -0,0 +1,21 @@
+export enum GetDashboard {
+  REQUEST = 'amundsen/dashboard/GET_DASHBOARD_REQUEST',
+  SUCCESS = 'amundsen/dashboard/GET_DASHBOARD_SUCCESS',
+  FAILURE = 'amundsen/dashboard/GET_DASHBOARD_FAILURE',
+}
+
+export interface GetDashboardRequest {
+  type: GetDashboard.REQUEST;
+  payload: {
+    uri: string;
+    searchIndex?: string;
+    source?: string;
+  }
+}
+
+export interface GetDashboardResponse {
+  type: GetDashboard.SUCCESS | GetDashboard.FAILURE;
+  payload: {
+    dashboard: any;
```

Comment:
nit: Add a `TODO` to update the type so we don't forget to come back to it once we know what the data will look like